current/runtime/events.md

```diff
@@ -15,16 +15,15 @@ when those events are emitted.
 Runtime events are created with the `decl_event!` macro.
 
 ```rust
-decl_event!{
-	pub enum Event<T> where
-		AccountId = <T as system::Trait>::AccountId,
-	{
-		ValueSet(u64, AccountId),
+decl_event!(
+	pub enum Event<T> where AccountId = <T as Trait>::AccountId {
+		/// Set a value.
+		ValueSet(u32, AccountId),
 	}
-}
+);
```

Comment:
`decl_event!` uses curly braces throughout Substrate repo. I tried to change pallet_collective to use the parens and semicolon, and it doesn't compile this way.

Reply:
Nevermind, The sudo pallet does it exactly as shown here, and it does compile fine as long as there is no space between the `!` and the `(`.